core/src/main/java/com/datastax/oss/driver/api/core/metadata/Node.java

```diff
@@ -71,10 +89,13 @@
   Optional<InetSocketAddress> getBroadcastAddress();
```

Comment:
It's too late obviously but I wonder why we went for optionals for some getters and nullable values for others.